docs/en/operations/server-configuration-parameters/settings.md

```diff
@@ -32,6 +32,39 @@ Type: UInt32
 
 Default: `120`
 
+## asynchronous_metric_log
+
+It is enabled by default on production deployment. 
```

Comment:
It's better to be more specific regarding what you mean by `production`, for example:
```suggestion
It's enabled by default on ClickHouse cloud deployments. 
```

Reply:
FWIW, I pulled and ran `24.12` ClickHouse docker to double check. It doesn't seem to be specific to cloud/production environment and seems to be enabled by default. You might want to double check this one.
https://pastila.nl/?0052096c/71c6aa4f1def1a52bd26d3b6abb9166f#IDvTnHCu/HNvc48dkGlCKA==

Reply:
Good point. 

What I found is that when using the quick install approach the asynchronous metric log are not enabled by default: https://clickhouse.com/docs/en/install#quick-install

But when using Production deployment approach it is enabled: https://clickhouse.com/docs/en/install#available-installation-options

I can say it is enabled by default and add a note to specify that using quick install, it is not.

Reply:
`nit:` Maybe it's just better to say that:

It's enabled by default on ClickHouse cloud deployments.
 
And be a bit generic about the `Note`. Somewhere along the lines:

For other environments, if it's not enabled by default, depending on how ClickHouse was installed or setup, you can manually enable or disable it as follows..